cert/aziot-certd/openapi/2020-09-01.yaml

```diff
@@ -0,0 +1,128 @@
+# Ref: https://spec.openapis.org/oas/v3.0.3
+
+openapi: '3.0.3'
+
+
+info:
+  title: 'aziot-certd API specification'
+
+  version: '2020-09-01'
+
+  description: |
+    This is the specification of the HTTP API of the aziot-certd service.
+
+  license:
+    name: 'Apache 2.0'
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+
+
+servers:
+- url: 'http://foo/'
```

Comment:
I believe it's valid to specify unix:// as the scheme, so `unix://run/aziot/certd.sock`.

Reply:
No, because the spec says about Path objects that:

>The path is appended to the URL from the Server Object in order to construct the full URL.